docs/explanation/core/configuration.md

```diff
@@ -4,746 +4,9 @@ order: 3
 
 # Configuration
 
-CometBFT can be configured via a TOML file in
-`$CMTHOME/config/config.toml`. Some of these parameters can be overridden by
-command-line flags. For most users, the options in the `##### main base configuration options #####` are intended to be modified while config options
-further below are intended for advance power users.
+CometBFT can be configured via a TOML file in `$CMTHOME/config/config.toml`.
 
-## Options
+Some of the parameters can be overridden by command-line flags.
 
-The default configuration file created by `cometbft init` has all
```

Comment:
Did you compare the text here with the new one in the CometBFT Configuration Manual? Is there anything in this text worth including there? Are the hunks toward the end of the PR doing this?